lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -21,6 +21,12 @@ end)
 
 __e2setcost(5) -- temporary
 
+e2function string entity:getUserGroup()
+	if not IsValid(this) then return self:throw("Invalid entity!", 0) end
+	if not this:IsPlayer() then return self:throw("Expected a Player but got an Entity!", 0) end
```

Comment:
These should return empty strings instead of 0.

Reply:
corrected